src/data/markdown/docs/02 javascript api/11 k6-ws/20-Params.md

```diff
@@ -6,10 +6,12 @@ excerpt: 'Used for setting various WebSocket request-specific parameters such as
-| --------------------- | ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-| `Params.headers`      | object | Object with key-value pairs representing custom headers the user would like to add to the request.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
-| `Params.tags`         | object | Key-value pairs where the keys are names of tags and the values are tag values. Response time metrics generated as a result of the request will have these tags added to them, allowing the user to filter out those results specifically, when looking at results data.                                                                                                                                                                                                                                                                                                                          |
+| Name                  | Type   | Description |
+| --------------------- | ------ | ----------- |
+| `Params.compression`  | string | Empty by default, but when specified, WebSocket compression will be enabled with the specified algorithm. The only currently supported algorithm is `deflate`. |
+| `Params.jar`          | [http.CookieJar](https://k6.io/docs/javascript-api/k6-http/cookiejar/) | The cookie jar that will be used when making the initial HTTP request to establish the WebSocket connection. If empty, the [default VU cookie jar](https://k6.io/docs/javascript-api/k6-http/cookiejar-method) will be used. |
+| `Params.headers`      | object | Object with key-value pairs representing custom headers the user would like to add to the request. |
+| `Params.tags`         | object | Key-value pairs where the keys are names of tags and the values are tag values. Response time metrics generated as a result of the request will have these tags added to them, allowing the user to filter out those results specifically, when looking at results data. |
```

Comment:
Key-value pairs where the keys are names of tags and the values are tag values. Response time metrics generated from the request have these tags attached, allowing the user to filter the results data.

Reply:
> Key-value [custom metric tags](https://k6.io/docs/using-k6/tags-and-groups/#user-defined-tags), where the keys are names of tags and the values are tag values. [Metrics samples generated from the WebSocket connection](https://k6.io/docs/javascript-api/k6-ws/socket/#websocket-built-in-metrics) will have these tags attached, allowing users to filter the results data or set [thresholds on sub-metrics](https://k6.io/docs/using-k6/thresholds/#thresholds-on-tags).

:question:

Reply:
[Custom metric tags](https://k6.io/docs/using-k6/tags-and-groups/#user-defined-tags) in key-value pairs where the keys are names of tags and the values are tag values. [The WebSocket connection will generate metrics samples](https://k6.io/docs/javascript-api/k6-ws/socket/#websocket-built-in-metrics) with these tags attached, allowing users to filter the results data or set [thresholds on sub-metrics](https://k6.io/docs/using-k6/thresholds/#thresholds-on-tags).

Reply:
Resolved by https://github.com/grafana/k6-docs/pull/495/commits/6f14bf3b21b18371e8a6a7aa8759b11a65e3c225